### ISSUE TITLE
Добавь скриншоштики в ридми пажалста<3

### ISSUE THREAD

**Issue by 4qiz** [Meresk/Kinopoisk-lite#1]
[sticker.webm](https://github.com/Meresk/Kinopoisk-lite/assets/89397489/35326adb-39b9-4131-8dab-0036a21acc37)

**Comment by SH4R1K** [comment]
Согласен! Мне тоже скриншотов не хватило🍻🍻🍻🍻🍻![7Y5Rb5IhnsI.jpg](https://github.com/Meresk/Kinopoisk-lite/assets/45074944/15d2a224-ee31-490a-b615-1461457377f9)
